src/public/modules/forms/admin/controllers/edit-fields-modal.client.controller.js

```diff
@@ -30,6 +30,26 @@ angular
     EditFieldsModalController,
   ])
 
+app.directive('validateEmailDomainFromText', () => {
```

Comment:
A little annoying, but can we separate the validator into its own directive (e.g. `validate-email-domain.client.directive.js`)

See: `src/public/modules/forms/base/directives/validate-url.client.directive.js`

You will need to also `require` it in `main.js`.